setup.py

```diff
@@ -9,8 +9,11 @@
     sys.stderr.write("ERROR: You need Python 3.2 or later to use mypy.\n")
     exit(1)
 
-from distutils.core import setup
-from distutils.command.build_py import build_py
+# This requires setuptools when building; setuptools is not needed
+# when installing from a wheel file (though it is still neeeded for
+# alternative forms of installing, as suggested by README.md).
+from setuptools import setup
```

Comment:
I am talking nonsense. What happens is as follows:
- the XML report template in `test-data/unit/cmdline.test` has $VERSION and $TIMESTAMP abstracted away because they will obviously be different too often
- the code responsible for it is in `mypy/test/testcmdline.py`, line 101
- however, with setuptools the test runner's mypy.__version__ is `"0.4.7-dev-2c9250eb182708402c66f8197f62ebe0190a43d5-dirty"` so it doesn't replace the `"0.4.7-dev"` string in the XML

I'm investigating now why this is the case.